### PR TITLE
OCS-336 and OCS-346 - Test ocs basics

### PR DESCRIPTION
**Pull request by shreekarSS**
resources/pod.py
 - Added function get_admin_key_from_ceph_tools(),which returns admin keyring in base64

ocs/defaults.py
 -  Added default path of csi templates of pvc and storage class for CEPHFS & RBD

ocs/constants.py
 - Added encode value for string 'admin'

tests/manage/OCS_336_346.py
- Added two cases OCS-336 and OCS-346, which covers basics such as secret creation, storage class creation with cephfs and rbd

resources/ocs.py
 - Added return statement to delete()

### CODE REVIEW

**Comment by shreekarSS** [comment]
http://pastebin.test.redhat.com/770553 --> test result log

File: tests/manage/OCS_336_346.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        self.cephfs_pvc = defaults.CSI_CEPHFS_PVC.copy()
+        pvc = OCS(**self.cephfs_pvc)
+        pvc.create()
+        PV = pvc.get()
+        assert 'Bound' == PV['status']['phase']
+        pvc.delete()
+        storage_class.delete()
+        secret.delete()
+
+    def test_ocs_346(self):
```
Comment (RazTamir) [inline tests/manage/OCS_336_346.py]: all comments apply to this test as well

File: tests/manage/OCS_336_346.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        secret.create()
+        self.cephfs_sc = defaults.CSI_CEPHFS_STORAGECLASS_DICT.copy()
+        self.cephfs_sc['parameters']['monitors'] = self.mons
+        self.cephfs_sc['parameters']['pool'] = (
+            f"{self.fs_data['metadata']['name']}-data0"
+        )
+        storage_class = OCS(**self.cephfs_sc)
+        storage_class.create()
+        self.cephfs_pvc = defaults.CSI_CEPHFS_PVC.copy()
+        pvc = OCS(**self.cephfs_pvc)
```
[the remark this answers is not in the document]
Comment (shreekarSS) [reply tests/manage/OCS_336_346.py]: Used PVC() class for creating pvc

File: tests/manage/OCS_336_346.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        pvc.create()
+        PV = pvc.get()
```
[the remark this answers is not in the document]
Comment (shreekarSS) [reply tests/manage/OCS_336_346.py]: Thanks, Updated

File: tests/manage/OCS_336_346.py [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+        assert 'Bound' == PV['status']['phase']
+        pvc.delete()
```
[the remark this answers is not in the document]
Comment (shreekarSS) [reply tests/manage/OCS_336_346.py]: Test case is all about creating and deleting resources.

File: tests/manage/OCS_336_346.py [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+        Testing basics: secret creation,
+        storage class creation and pvc with cephfs
+        """
+        self.cephfs_secret = defaults.CSI_CEPHFS_SECRET.copy()
+        del self.cephfs_secret['data']['userID']
+        del self.cephfs_secret['data']['userKey']
+        self.cephfs_secret['data']['adminKey'] = (
+            get_admin_key_from_ceph_tools()
+        )
+        self.cephfs_secret['data']['adminID'] = defaults.constants.ADMIN_BASE64
```
Comment (RazTamir) [inline tests/manage/OCS_336_346.py]: import constants and use it directly

Comment (shreekarSS) [reply tests/manage/OCS_336_346.py]: Updated now with import

**Comment by shreekarSS** [comment]
http://pastebin.test.redhat.com/770851 --> test logs

**Pull request merged**